examples/CRISP/apps/client/src/context/voteManagement/VoteManagement.context.tsx

```diff
@@ -100,6 +101,15 @@ const VoteManagementProvider = ({ children }: VoteManagementProviderProps) => {
     setIsLoading(false)
   }, [wasmLoading, enclaveLoading])
 
+  // Update user state when wallet connection changes
+  useEffect(() => {
+    if (isConnected && address) {
+      setUser({ address })
+    } else {
+      setUser(null)
+    }
+  }, [isConnected, address, setUser])
```

Comment:
This is fine but technically I don't believe `setUser` will ever change it's reference.

Reply:
yeah it wont since it comes from a useState hook 👍